index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr. Nabeel</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.pixabay.com/photo/2015/01/08/18/29/entrepreneur-593358_960_720.jpg"
+          alt="Dr.Nabeel sitting"
+        />
+        <figcaption id="img-caption">we are good</figcaption>
+      </div>
+      <section id="tribute-info">
+        <h3>Here's a timeline of Dr. Nabeel's life</h3>
+        <ul>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
+        </ul>
+      </section>
+      <button>
+        <a
+          id="tribute-link"
+          href="https://github.com/PeteUgwu/nabel-and-peter-.git"
+          target="_blank"
+          >see more info</a
+        >
+      </button>
+    </main>
+    <script src="js/index.js"></script>
```

Comment:
Please use "Class Selector" more often.  You have used  "ID selector" everywhere. IDs should be used for unique items on the page. 😊😊😊😎